docs/reference/language/scripting.md

```diff
@@ -7,6 +7,16 @@ Typst embeds a powerful scripting language. You can automate your documents and
 create more sophisticated styles with code. Below is an overview over the
 scripting concepts.
 
+## Identifiers
```

Comment:
Not sure if this change belongs in this PR, but I'll let @laurmaedje have the final word on this.

Reply:
Not sure if it is worth moving this to a PR of its own. In any case, this is another place where it is necessary to unlearn experience from other programming languages, and therefore important to mention.

Reply:
I think it would be good to move this.